EC-Switch-Seclog.py

```diff
@@ -0,0 +1,973 @@
+#!/usr/bin/python -u
+import signal
+import sys, getopt
+import subprocess, pkg_resources
+import os
+import subprocess
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Consider possible security implications associated with subprocess module. https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html#b404-import-subprocess